sample/src/main/java/ffmpeg/videokit/sample/MainActivity.java

```diff
@@ -71,9 +71,9 @@ public void onMediaFileSelected(String path) {
 
         final Command command = videoKit.createCommand()
                 .overwriteOutput()
-                .addInputPath(path)
+                .inputPath(path)
                 .outputPath(path + POSTFIX)
-                .addCustomCommand("-ss 1 -t 3")
+                .customCommand("-ss 1 -t 3")
```

Comment:
It's just example command. Not sure if we should move CLI command to static field in this case. I afraid that in this particular case usage of static constant will lead to the lose of context for reader :)